test/image/mocks/zzz_shape_shift_horizontal.json

```diff
@@ -0,0 +1,87 @@
+{
+    "data": [
+        {
```

Comment:
Let's add a `scatter` and a `scattergl` trace with several points to test if they are selected properly.

Reply:
I added a scatter trace to shape_shift_vertical where there is also a selection.
When I tried to add scattergl, it wasn't selected. But this was also the case when I tried it out on master (same mock, with type "scattergl" instead of "scatter"). Am I missing something?